stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -146,7 +147,7 @@ internal class PaymentSheetViewModel internal constructor(
         customerId: String,
         stripeAccountId: String? = this.stripeAccountId
     ) {
-        viewModelScope.launch {
+        CoroutineScope(workContext).launch {
```

Comment:
Can we launch on `viewModelScope` but switch to `workContext`?

```
viewModelScope.launch {
    withContext(workContext) {
    }
}
```

Reply:
sure